v15

WilliamDenniss opened the pull request.

WilliamDenniss committed:
v15

WilliamDenniss committed:
Renamed spec.

Edit pass.

WilliamDenniss committed:
Date update and tweaks

selfissued commented on draft-ietf-oauth-device-flow.xml:
```diff
@@ -649,11 +654,12 @@
-  to the OAuth client. Only a one-way channel is required to make this flow
-  useful in many scenarios. For example, an HTML application on a TV that can
-  only make outbound requests. If a return channel were to exist for the chosen user
+  to device with the OAuth client. This protocol only requires a one-way
```
I suspect that the wording "communicate back to device with the OAuth client" needs to be corrected.

WilliamDenniss replied:
Ah, missed this comment. Have staged a change for the next release it now reads "communicate back to the device with the OAuth client"

selfissued reviewed (changes requested):
Please correct the wording problem identified.  After that, I'll be good with merging this.

WilliamDenniss committed:
Added explicit text regarding client authentication

Thank you Brian Grant, Dan McNulty and Jamshid Khosravian for reporting this.

WilliamDenniss committed:
Documented error response for the authorization endpoint

Corrected usability considerations.

Thank you to Emond for reporting these issues

WilliamDenniss committed:
Revised usage of the term "flow"

Edit pass

WilliamDenniss committed:
Add Filip

WilliamDenniss committed:
Typos

WilliamDenniss committed:
Improved ASCII diagram
